app/src/main/AndroidManifest.xml

```diff
@@ -49,6 +49,7 @@
         android:label="@string/app_title"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:theme="@style/MainTheme"
+        android:networkSecurityConfig="@xml/network_security_config"
```

Comment:
@TomTasche this was necessary to get HTTP working. without that we have to use HTTPS